### PR TITLE
Header sidebar profile page

### PR DESCRIPTION
**Pull request by zoef-yt**
[Files to review](https://github.com/zoef-yt/gamers-connect/pull/1/files)
In this Pr, I added,
UI for header, sidebar, and profile page 
[visit profile page](https://deploy-preview-1--gamers-connect.netlify.app/my-profile)

### CODE REVIEW

File: src/Components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+				<NavLink className='navbar-brand ' to='/'>
+					Gamers Connect
+				</NavLink>
+				<input type='search' className='text-field' placeholder='Search' />
+				<div className='navbar-cta'>
+					<div
+						onClick={() => (!isSideBarOpen ? openSideBar() : closeSideBar())}
+						className={`menu-burger ${isSideBarOpen && 'menu-burger-opened'}`}
+					>
+						<div></div>
```
Comment (santhoshpemmaka) [inline src/Components/Navbar/Navbar.jsx:25]: If possible remove the empty div tags. one small suggestion.

**Review by santhoshpemmaka — APPROVED** [review]

File: src/Context/UserContext/UserContext.jsx [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+const UserContext = createContext();
+
+const UserProvider = ({ children }) => {
+	const [user, setUser] = useState({});
+
+	const [userPosts, setUserPost] = useState([]);
+
+	useEffect(() => {
+		setUser({
+			displayName: 'Zoef Shaikh',
```
Comment (saurabhan) [inline src/Context/UserContext/UserContext.jsx:12]: I hope this is a temporary data placeholder and you'll be fetching from an API? or abstracting your own DB?

File: src/Assets/AllSVG.jsx [hunk 3]
```diff
@@ -0,0 +1,189 @@
+function SunIcon(props) {
```
Comment (saurabhan) [inline src/Assets/AllSVG.jsx:1]: Any reason for building custom icons rather than using prebuilt libs?

**Review by saurabhan — APPROVED** [review]
The animations look great. good looking UI
UI suggestion: When the sidebar is open and closed the icons do not align( the closed sidebar icons sit a little higher up than the open one) this causes a jitter effect, other than that everything else is good.

File: src/Context/UserContext/UserContext.jsx (hunk 2 above)
Comment (zoef-yt) [reply src/Context/UserContext/UserContext.jsx:12]: Yep this is a temp data.

File: src/Assets/AllSVG.jsx (hunk 3 above)
Comment (zoef-yt) [reply src/Assets/AllSVG.jsx:1]: Nope, I have been using custom icons so just continuing them

File: src/Components/Navbar/Navbar.jsx (hunk 1 above)
Comment (zoef-yt) [reply src/Components/Navbar/Navbar.jsx:25]: done.

Comment (zoef-yt) [reply src/Components/Navbar/Navbar.jsx:25]: These div's make the hamburger icon can't remove them.

**Pull request merged**